common-ui/src/main/res/values/design-system-theming.xml

```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
 <!--
```

Comment:
In line 68 (I can't add a comment there) the style should be `Secondary.Small` :

`<item name="secondaryButtonSmallStyle">@style/Widget.DuckDuckGo.Button.Secondary.Small</item>`